### ISSUE TITLE
Support Xcode 11.2 Swift 5.1.2

### ISSUE THREAD

**Issue by paradite** [alibaba/macaca#952]
```
Module compiled with Swift 5.1 cannot be imported by the Swift 5.1.2 compiler: /usr/local/lib/node_modules/app-inspector/node_modules/xctestwd/Carthage/Build/iOS/Swifter.framework/Modules/Swifter.swiftmodule/arm64-apple-ios.swiftmodule
```

**Comment by paradite** [comment]
fixed in https://github.com/macacajs/XCTestWD/pull/168

**Comment by carmensmile** [comment]
When can support Xcode 11.4 Swift 5.2?

**Comment by carmensmile** [comment]
> fixed in [macacajs/XCTestWD#168](https://github.com/macacajs/XCTestWD/pull/168)

When can support Xcode 11.4 Swift 5.2?

**Comment by paradite** [comment]
> > fixed in [macacajs/XCTestWD#168](https://github.com/macacajs/XCTestWD/pull/168)
> 
> When can support Xcode 11.4 Swift 5.2?

Yes. We will support Xcode 11.4 Swift 5.2 soon. Hopefully this weekend~
